script/deploy/mainnet/005_RegisterLidoWithdrawalsScript.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.23;
+
+import "forge-std/console.sol";
+import {Vm} from "forge-std/Vm.sol";
+
+import {IERC20, IWETH, LegacyAMM} from "contracts/Interfaces.sol";
+import {LidoARM} from "contracts/LidoARM.sol";
+import {CapManager} from "contracts/CapManager.sol";
```

Comment:
Not used.

Reply:
`LidoARM` is used. I've removed the others. Thanks